### PR TITLE
Used apt package for openvino installation

### PR DESCRIPTION
**Pull request by mohdansx**
1. OpenVINO is installed using the apt package
2. OpenCL drivers are installed seperately
3. Cmake version 3.13 is used
4. Final image size is 1.91GB

### CODE REVIEW

**Comment by smkarlap** [comment]
changed the target branch to 'master'

File: Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile [hunk 1]
```diff
@@ -50,30 +50,36 @@ ENV LD_LIBRARY_PATH=${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/exte
 ENV LANG en_US.UTF-8
 ENV PATH /opt/cmake/bin:$PATH
 
-COPY l_openvino_*.tgz .
+RUN mkdir ${MY_ROOT}/apt_key
+COPY GPG-PUB-KEY-INTEL-OPENVINO-2020 ${MY_ROOT}/apt_key/
```
Comment (smkarlap) [inline Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile:54]: can't we download this inside the container using a wget?

File: Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile [hunk 2]
```diff
@@ -36,9 +36,9 @@ ARG ONNXRUNTIME_REPO=https://github.com/microsoft/onnxruntime
 ARG ONNXRUNTIME_BRANCH=master
 
 ENV pattern="COMPONENTS=DEFAULTS"
-ENV replacement="COMPONENTS=intel-openvino-ie-sdk-ubuntu-xenial__x86_64;intel-openvino-ie-rt-cpu-ubuntu-xenial__x86_64;intel-openvino-ie-rt-gpu-ubuntu-xenial__x86_64;intel-openvino-ie-rt-vpu-ubuntu-xenial__x86_64;intel-openvino-ie-rt-hddl-ubuntu-xenial__x86_64;intel-openvino-model-optimizer__x86_64;intel-openvino-opencv-lib-ubuntu-xenial__x86_64"
+ENV replacement="COMPONENTS=intel-openvino-ie-sdk-ubuntu-xenial__x86_64;intel-openvino-ie-rt-cpu-ubuntu-xenial__x86_64;intel-openvino-ie-rt-gpu-ubuntu-xenial__x86_64;intel-openvino-ie-rt-vpu-ubuntu-xenial__x86_64;intel-openvino-ie-rt-hddl-ubuntu-xenial__x86_64;intel-openvino-opencv-lib-ubuntu-xenial__x86_64"
 ENV LD_LIBRARY_PATH=/opt/miniconda/lib:/usr/lib:/usr/lib/x86_64-linux-gnu:$LD_LIBRARY_PATH
-ENV INTEL_OPENVINO_DIR=/opt/intel/openvino_2020.2.120
+ENV INTEL_OPENVINO_DIR=/opt/intel/openvino_2020.2.130
```
Comment (smkarlap) [inline Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile:41]: Why the change from 120 to 130? is this a new version of openvino binary?

File: Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/README.MD [hunk 3]
```diff
@@ -73,12 +73,12 @@
...
 
     ```
-## VAD-R Accelerator Version 
+## VAD-M Accelerator Version 
 
 1. Retrieve your docker image in one of the following ways. 
    - Build the docker image from the DockerFile in this repository.
      ``` 
-      docker build -t onnxruntime-vadr --build-arg DEVICE=VAD-R_FP16 --network host . 
+      docker build -t onnxruntime-hddl --build-arg DEVICE=VAD-M_FP16 --network host .
```
Comment (smkarlap) [inline Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/README.MD:81]: why don't we name the docker image to 'onnxruntime-vadm' ? 'hddl' was an internal name and should no longer be used in documentation

File: Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile (hunk 2 above)
Comment (mohdansx) [reply Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile:41]: The available openvino apt package for 2020.2 was 130.
We checked using `sudo apt-cache search intel-openvino-dev-ubuntu16`

File: Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile (hunk 1 above)
Comment (mohdansx) [reply Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile]: Updated.
